app/parse.py

```diff
@@ -7,9 +14,67 @@ class Quote:
     author: str
     tags: list[str]
 
+    @classmethod
+    def parse_one_quote(cls, quote_soup: Tag) -> "Quote":
```

Comment:
Leave a classes in a return type annotation without `""` in the future
```suggestion
    def parse_one_quote(cls, quote_soup: Tag) -> Quote:
```